src/main/java/teammates/common/datatransfer/questions/FeedbackMsqQuestionDetails.java

```diff
@@ -191,9 +209,16 @@ public String getQuestionWithExistingResponseSubmissionFormHtml(
                         Slots.MSQ_CHOICE_TEXT, "<i>" + Const.NONE_OF_THE_ABOVE + "</i>");
         optionListHtml.append(optionFragment).append(Const.EOL);
 
+        boolean isMaxSelectableChoicesDisabled = maxSelectableChoices == Integer.MIN_VALUE;
+
         return Templates.populateTemplate(
                 FormTemplates.MSQ_SUBMISSION_FORM,
-                Slots.MSQ_SUBMISSION_FORM_OPTION_FRAGMENTS, optionListHtml.toString());
+                Slots.MSQ_SUBMISSION_FORM_OPTION_FRAGMENTS, optionListHtml.toString(),
+                Slots.DISABLED, isMaxSelectableChoicesDisabled ? "disabled" : "",
+                Slots.MSQ_PARAM_MAX_SELECTABLE_CHOICES, Const.ParamsNames.FEEDBACK_QUESTION_MSQ_MAX_SELECTABLE_CHOICES,
+                Slots.QUESTION_INDEX, Integer.toString(qnIdx),
+                Slots.MSQ_MAX_SELECTABLE_CHOICES,
+                        Integer.toString(isMaxSelectableChoicesDisabled ? 2 : maxSelectableChoices));
```

Comment:
Yes, there is no limit if this field is disabled. I put `2` as a default value. It has no significance. Will remove it.